src/ProgressOnderwijsUtils/RandomHelper.cs

```diff
@@ -113,10 +113,12 @@ public string GetStringOfUriPrintableCharacters(int length)
             return new string(Enumerable.Range(0, length).Select(_ => UriPrintableCharacters[GetUInt32((uint)UriPrintableCharacters.Length)]).ToArray());
         }
 
+#if NET461

```

Comment:
of we fixen deze implementatie om de toch al bestaande random-string functionaliteit te gebruiken die inzichtelijker veilig is.

Reply:
ook prima.